tests/test_config.py

```diff
@@ -382,3 +399,99 @@ def test_noenv_nodef_invalid_setup(self):
                 pass
             else:
                 self.fail("Expected TomlDecodeError from invalid config file")
+
+
+class get_ipython_profile_from_config_TestCase(unittest.TestCase):
+    """
+    Test cases for function `get_ipython_profile_from_config`.
+    """
+
+    def setUp(self):
+        """
+        Set fixtures for this test case.
+        """
+        set_config_files_fixture(self)
+
+    def test_missing_key_setup(self):
+        """
+        Setup: $IPDB_CONFIG unset, $HOME/.ipdb does not exist,
+            setup.cfg does not exist, pyproject.toml content is invalid.
+        Result: Propagate exception from `get_config`.
+        """
+        os.unlink(self.env_filename)
+        os.unlink(self.default_filename)
+        os.unlink(self.setup_filename)
+        write_lines_to_file(
+            self.pyproject_filename,
+            ["[tool.ipdb]"],
+        )
+
+        try:
```

Comment:
Am I right to think that those imports are left over from copy paste or I am missing sthing ?